### PR TITLE
Cleanup protoypye 2

### PR DESCRIPTION
**Pull request by knollfear**
Resolves #

**Description-**

Opens Subpart when the blue button for that subpart is clicked
Populates section tab with all data for that section on load

**This pull request changes...**

- expected change 1

**Steps to manually verify this change...**

in PD view clicking on a blue button for a subpart will switch to that section and expand it
sections tab should load content on page load.

### CODE REVIEW

**Review by PhilR8 — APPROVED** [review]

File: solution/backend/supplemental_content/views.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def get(self, *args, **kwargs):
             locations__title=title,
             locations__part=part,
         )
-        if len(section_list) > 0 or len(subpart_list) > 0 and len(subjgrp_list) > 0:
+        if len(section_list) > 0 or len(subpart_list) > 0 or len(subjgrp_list) > 0:
```
Comment (knollfear) [inline solution/backend/supplemental_content/views.py:72]: this bug made it to prod but this will fix it.

Comment (thwalker6) [reply solution/backend/supplemental_content/views.py:72]: looks fine.

**Review by thwalker6 — APPROVED** [review]
Everything looks good and works it seems.

**Review by cgodwin1 — APPROVED** [review]
LGTM too!

**Pull request merged**